### PR TITLE
MIG-1518 Wrong Must Gather for MTC

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTC 1.8.3

OADP 1.3.1

OCP 4.13 +

Resolves - 

https://issues.redhat.com/browse/MIG-1518

https://issues.redhat.com/browse/OADP-3470

Deploy preview - 

OADP

https://74506--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#migration-using-must-gather_oadp-troubleshooting

Migrating from 3-4

https://74506--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-using-must-gather_troubleshooting-3-4

MTC

https://74506--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/troubleshooting-mtc.html#migration-using-must-gather_troubleshooting-mtc

### CODE REVIEW

**Review by stillalearner — APPROVED** [review]
lgtm

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/migration-using-must-gather.adoc [hunk 1]
```diff
@@ -38,10 +41,24 @@ endif::[]
 
 ifdef::troubleshooting-3-4,troubleshooting-mtc[]
 * To collect data for the past hour:
++
+[source,terminal]
+----
+oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.8
```
Comment (skrthomas) [inline modules/migration-using-must-gather.adoc:47]:
```suggestion
$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.8
```

**Review by skrthomas — COMMENTED** [review]
A couple of places where the $ is missing from the oc command.

**Comment by CarmiWisemon** [comment]
@skrthomas 
Thank you for your review!
Carmi

**Pull request merged**